samples/golang/mywebapi/src/app/hello.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+)
+
+func sayHelloHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Hello")
+}
+
+func pingHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
If the purpose of this endpoint is to give something for probes to check, the conventional route for this in Go is `/healthz`.

Reply:
are you saying to replace the "/api/ping" to "/healthz" ?

Reply:
yes